src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -14,6 +28,30 @@
 
     @Override
     public int getSize() {
-        return 0;
+        return size;
+    }
+
+    private int getHash(K key) {
```

Comment:
The `getHash` method is defined as private and it's fine because it's a utility method used only inside the class. But it's not used anywhere. You should use it in your `put` and `getValue` methods to determine the index of the bucket where to put or from where to get the node.